Tasks/AzureTestPlanV0/Invokers/gradleinvoker.ts

```diff
@@ -26,5 +25,5 @@ export async function executegradletests(testsToBeExecuted: string[]) {
 
     var status = await execGradleBuild(args);
```

Comment:
how are the exceptions occuring inside execGradleBuild being handled?

Reply:
Please check testLibExecutor.ts file. We are logging the error and returning the status 1